src/lib/isMobilePhone.js

```diff
@@ -41,6 +41,7 @@ const phones = {
   'en-KE': /^(\+?254|0)(7|1)\d{8}$/,
   'en-MT': /^(\+?356|0)?(99|79|77|21|27|22|25)[0-9]{6}$/,
   'en-MU': /^(\+?230|0)?\d{8}$/,
+  'en-NA': /^(\+?264|0)(6|8)\d{7}$/,
```

Comment:
Isn't BF's locale `fr-BF`? https://www.localeplanet.com/icu/fr-BF/index.html

Reply:
scratch that, seen your changes below; perhaps you can have this change as separate PR.

Reply:
Ok, never mind, I'll just update the title of the PR to include both Namibia and BF.